README.md

```diff
@@ -16,7 +16,22 @@ understand how everything fits together.
 This document guides you through creating a highly available OpenShift
 cluster on AWS.
 
-## Labs
+## Documentation
+With the third beta drop of OpenShift 4, this repository will link you to the work-in-progress official documentation. Currently that documentation is behind a user/password (for anti-search-engine-indexing purposes.
```

Comment:
I think that you won't need a password after the Beta 3 drop.

Reply:
@kalexand-rh - a password will be required till GA of 4.x.

@thoraxe